packages/tasks/src/model-libraries-snippets.ts

```diff
@@ -46,6 +46,18 @@ export const asteroid = (model: ModelData): string[] => [
 model = BaseModel.from_pretrained("${model.id}")`,
 ];
 
+export const audioseal = (model: ModelData): string[] => [
+	`from audioseal import AudioSeal
+
+model = AudioSeal.load_generator("${model.id}")
+
+wav, sr = tensor_wav, 16000
```

Comment:
Could we update to show what `tensor_wav` is please?

Reply:
Fixed in the latest commit.